requirements.txt

```diff
@@ -5,7 +5,11 @@ modelmeta
 shapely
 numpy
 netcdf4
-GDAL
+# nchelpers
+GDAL<2.0 # Allow pip or system install for travis
 rasterio
+cdo
 # For testing
+pytest
 python-dateutil
+pip
```

Comment:
pip is kind of assumed to exist. Should not be included in requirements.txt. I.e. how would you "pip install -r requirements.txt" if you didn't have pip installed?